tests/TODO.md

```diff
@@ -186,6 +186,20 @@ groups.
 * **Effective space**. Veryfy calculations of `get_free_space()` and
   `get_effective_space()`. Test cases are the same as for Couple's test
   **Effective space**. **TODO**: docs.
+* **History**. Verify applying of `GroupHistoryEntry` coming from history
+  database. If some of group backends are absent in history entry, they
+  must be removed from Group's set of backends. Actions are performed in
```

Comment:
This concerns only non-automatic history entries. History entries of type 'automatic' should not be applied by workers.